app/Http/Controllers/Api/Remote/Servers/ServerInstallController.php

```
@@ -15,14 +16,12 @@ class ServerInstallController extends Controller
     /**
      * Returns installation information for a server.
      */
-    public function index(Server $server): JsonResponse
+    public function index(ServerRequest $request, Server $server): JsonResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)